pyproject.toml

```diff
@@ -22,18 +22,18 @@ build = "build.py"
 [tool.poetry.dependencies]
 cmake = "~3.18.0"
 importlib-metadata = ">=4.4,<5.0"
-numpy = "^1.18.0"
-python = ">=3.7,<3.9"
-tensorflow = "~2.4.0"
+numpy = "^1.21.0"
+python = ">=3.10,<3.11"
+tensorflow = "~2.8.0"
```

Comment:
This is a very old version of `tensorflow`. Now the build is running it would be worth investigating how close to the current version we can get.

Reply:
I have played further with higher versions